static/app/views/issueList/draggableTabBar.tsx

```diff
@@ -64,6 +60,11 @@ export interface DraggableTabBarProps {
    * Callback function to be called when user clicks the 'Save View' button for temporary views.
    */
   onSaveTempView?: () => void;
+  /**
+   * Callback function to be called when user renames a tab.
+   * Note: The `Rename` button only appears for persistent views
+   */
+  onTabRenamed?: (key: MenuItemProps['key'], newLabel: string) => void;
```

Comment:
Changed `onRename` -> `onTabRenamed` because the parent component probably cares when the tab has already been renamed, not when the `Rename` menu option is clicked.